src/popup/popup.js

```diff
@@ -38,19 +49,29 @@ function onWindowLoad() {
 function showLoading() {
     document.getElementById("loading-image").style.display = "block";
     document.getElementsByClassName("button-container")[0].style.display="none";
+    document.getElementById("error-message").style.display = "none"; // adding to avoid any risks!
 }
 
 function removeLoading() {
     document.getElementById("loading-image").style.display = "none";
     document.getElementsByClassName("button-container")[0].style.display = "block";
+    document.getElementById("error-message").style.display = "none"; // adding to avoid any risks!
 }
 
 function injectTimetable() {
     showLoading();
 	chrome.tabs.executeScript(null, {
 		file: "/src/timetable/activateTimetable.js"
-	}, function() {
-		if (chrome.runtime.lastError)	console.log(chrome.runtime.lastError.message);
+    }, function () {
+        if (chrome.runtime.lastError) {
+            switch (chrome.runtime.lastError.message) {
+                case "Cannot access a chrome:// URL":
```

Comment:
I could not find error code in the `chrome.runtime.lastError`. Let me know if there is a better way to do this.

Reply:
This looks fine. However, I don't see the need for both an `if` and a `switch` here. Maybe you can do some refactoring?